docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,62 @@
+
+To integrate Codacy with your Git workflow, follow these steps:
+
+1.  [Configuring the quality gate rules](#configuring-gate)
+1.  [Activating the Git provider integration](#git-provider-integration)
+1.  [Blocking merging pull requests](#blocking-pull-requests) (optional)
+
+## 1. Configuring the quality gate rules {: id="configuring-gate"}
+
+[Review and adjust the quality settings](../repositories-configure/adjusting-quality-settings.md) of your repository to decide which pull requests should fail the Codacy quality gate.
+
+!!! tip
+    The default values of the quality gate rules are designed to help maintain the current code quality of your repository. In particular, the default value for the coverage rule might be demanding. Depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of delaying or blocking the development progress.
+
+    Codacy generally recommends that on a first stage you configure rules that focus on stopping new critical issues from entering your code base, such as:
+
+    -   High severity issues
+    -   Security issues
+    -   Considerable drops in code coverage
+
+!!! important
+    **If you want to use code coverage** to block merging pull requests that don't meet your standards, make sure that you enable the rule **Diff coverage is under** or **Coverage variation is under**. This is required for Codacy to report the coverage status directly on your pull requests.
+
+![Adjusting the quality settings](../repositories-configure/images/quality-settings.png)
+
+## 2. Activating the Git provider integration {: id="git-provider-integration"}
+
+Follow the instructions for [GitHub](../repositories-configure/integrations/github-integration.md#enabling), [GitLab](../repositories-configure/integrations/gitlab-integration.md#enabling), or [Bitbucket](../repositories-configure/integrations/bitbucket-integration.md#enabling) depending on your Git provider, and make sure that you:
+
+1.  Enable the Git provider integration
+1.  Enable the option **Status checks** (GitHub) or **Pull request status** (GitLab and Bitbucket)
+
+![Enabling your Git provider integration](../repositories-configure/integrations/images/github-integration.png)
+
+## 3. Blocking merging pull requests (optional) {: id="blocking-pull-requests"}
+
+Once you've tested out Codacy for a while and you're happy with the level of feedback provided, you can decide to enforce the quality gates and use Codacy to block merging pull requests on your Git provider. This is the best way to protect your code from unwelcome changes and fully integrates code analysis into your development pipeline.
```

Comment:
An alternative here:
```suggestion
Once you've tested out Codacy for a while and you're happy with the level of feedback provided, you can decide to enforce the quality gates and use Codacy to block merging pull requests on your Git provider. This is the best way to protect your code from unwelcome changes and fully integrates code quality and coverage analysis into your development pipeline.
```

Reply:
Good idea, I guess we need to start being more explicit about quality [and|or|versus] coverage. :+1: